cmd/virt-launcher-monitor/virt-launcher-monitor.go

```diff
@@ -55,7 +55,7 @@ func cleanupContainerDiskDirectory(ephemeralDiskDir string) {
 	}
 }
 
-func cleanupSerialConsoleSock(uid string) {
+func createSerialConsoleTermFile(uid string) {
 	// Create a file to quickly signal the shutdown to the guest-console-log container in the case the sigterm signal got
```

Comment:
This comment is now outdated, please change it.

Reply:
done